src/core/modules/StarWrapper.cpp

```diff
@@ -324,9 +330,14 @@ QString StarWrapper1::getInfoString(const StelCore *core, const InfoStringGroup&
 		{
 			QString mmStr = q_("Rising time");
 			if (ebsFlag)
+			{
 				mmStr = q_("Duration of eclipse");
-
-			oss << QString("%1: %2%").arg(mmStr).arg(vMm) << "<br />";
+				float durDays = vPeriod * vMm / 100.0;
```

Comment:
This part of code may be written more compact

Reply:
sorry, I don't see exactly what you mean?

please note that I am a big believer of easily readable lines of code with copious use of whitespace
rather than compact - and sometimes cryptic - code (think of Perl-like scripts...)

to give an example:

![image](https://user-images.githubusercontent.com/3529789/98466783-05421e00-21d2-11eb-803c-84ab9c0073e6.png)

Reply:
What about: 
```
if (ebsFlag)
	oss << QString("%1: %2% (%3)").arg(q_("Duration of eclipse")).arg(vMm).arg(StelUtils::daysFloatToDHMS(vPeriod * vMm / 100.f)) << "<br />";
else
	oss << QString("%1: %2%").arg(q_("Rising time")).arg(vMm) << "<br />";
```

> please note that I am a big believer of easily readable lines of code with copious use of whitespace rather than compact - and sometimes cryptic - code (think of Perl-like scripts...)

No problem, but this code shouldn't introduce real and potential problems (hint: type cast)

Reply:
no, definitely not. the line is far too long to be readable, even if it works fine.

as you can see in the screenshot below, my editor is already complaining :-/ (red line = column 80)

![image](https://user-images.githubusercontent.com/3529789/98468779-4ccda780-21dc-11eb-80fe-c3a066ac099f.png)